rails-buildpack/3.x/Dockerfile

```
@@ -33,6 +33,7 @@ RUN apt-get update \
       gstreamer1.0-tools \
       gstreamer1.0-x \
       imagemagick \
+ && apt-get upgrade -y \
```

I think it's better to use unattended-upgrade which upgrades package in a minimum way.

```suggestion
     unattended-upgrades \
 && unattended-upgrade \
```

---

After my quick research (basically asking LLM 😓 ), `unattended-upgrade` is usually used for a daemon process on the running server to avoid including breaking changes, and it's not common to use it in Dockerfile. `upgrade` is recommended to apply all security patches. (and if it's broken, we can catch the issue in E2E or review app).
So, I'll use `upgrade`.

Anyway, thank you for your suggestion, `unattended-upgrades` is TIL.